package.json

```diff
@@ -62,6 +62,7 @@
     "prepare": "husky install"
   },
   "lint-staged": {
+    "*": "cspell",
```

Comment:
I think we need `--no-must-find-files` here

Reply:
```suggestion
    "*": "cspell --no-must-find-files",
```